packages/shared-state/files/usr/lib/lua/shared-state.lua

```diff
@@ -148,7 +148,6 @@ function SharedStateBase:sync(urls)
 	self:unlock()
 	self:_sync(urls)
 	self:lock()
-	self:load() -- Take in account changes happened during sync
```

Comment:
@spiccinini , thanks for catching this issue!

But together with the change at load method, we are changing the behavior of sync.
In master version, changes to the local storage made by another process during sync are merged with those coming from the sync process. I think that's what "Take in account changes happened during sync" means.
In this branch version, we are ignoring those changes.
If shared-state-publish_dnsmasq_leases gets executed during sync for instance, and the host has given new leases, those will enter shared state storage, then leave when sync finish, and then come back when publish happens again.
I think the most safe option is to keep the current behavior of the master branch.
Thoughts?

Reply:
This is fixed now.